docs/core/linux-prerequisites.md

```diff
@@ -19,24 +19,53 @@ This article shows the dependencies needed to develop .NET Core applications on
 
 <!-- markdownlint-disable MD025 -->
 
+# [.NET Core 3.0](#tab/netcore30)
+
+.NET Core 3.0 treats Linux as a single operating system. There is a single Linux build (per chip architecture) for supported Linux distributions. 
+
+For download links and more information, see [.NET Core 3.0 downloads](https://dotnet.microsoft.com/download/dotnet-core/3.0).
+
+.NET Core 3.0 is supported on the following Linux distributions/versions:
+
+| OS                             | Version               | Architectures    |
+| ------------------------------ | --------------------- | ---------------- |
+| Red Hat Enterprise Linux       | 6, 7                  | x64 |
+| Oracle Linux                   | 7                     | x64 |
+| CentOS                         | 7                     | x64 |
+| Fedora                         | 28, 29, 30            | x64 |
+| Debian                         | 9, 10                 | x64, ARM32, ARM64 |
+| Ubuntu                         | 16.04 or later, 18.04 or later, 19.04 or later | x64, ARM32, ARM64 |
+| Linux Mint                     | 18                    | x64 |
+| OpenSUSE Leap                  | 42.3 or later         | x64 |
```

Comment:
This is not listed in https://github.com/dotnet/core/blob/master/release-notes/3.0/3.0-supported-os.md. Instead it lists openSUSE 15+